scripts/s3-dist.sh

```diff
@@ -31,7 +31,7 @@ function cleanup(){
 }
 
 echo Installing dependencies
-npm install -g serverless && npm install
+npm install -g serverless@2.64.1 && npm install
```

Comment:
I think that line 43 below will not work without serverless globally installed.

Reply:
Agreed I will leave it in but pinned to 2.64.1